### ISSUE TITLE
Support applying tags when creating items

### ISSUE THREAD

**Issue by zcutlip** [zcutlip/pyonepassword#83]
It should be possible to create items with tags pre-applied.

the `op item create` subcommand has a `--tags` argument that takes a comma separated list of tags to apply. It may also be possible to add the tags directly to the JSON template.

The API should look something like:

```python
  def item_create(self,
                    new_item: OPNewItemMixin,
                    password_recipe: OPPasswordRecipe = None,
                    tags: List[str] = [],
                    vault: Optional[str] = None):
```

**Comment by zcutlip** [comment]
This is currently working in the branch `dev/83-support-applying-tags-when-creating-items`

Rather than using the `--tags` cli arg, we're adding a `tags` field to the new item dictionary.

**Comment by zcutlip** [comment]
I'm going to merge this work-in-progress branch into `development` in order to make use of it in testing #82

**Comment by zcutlip** [comment]
addressed by #91 & #94
